### ISSUE TITLE
Blocked editing to Public Folder...

### ISSUE THREAD

**Issue by irishburlybear** [solid/solid#159]
I wrote a simple index.html file with a test div in it just to see if I can write html code. Now I can't get back to the editor tools. :(

**Comment by RubenVerborgh** [comment]
That's right, the default index page is only served when there is no index.html in there.

You can delete your custom index file to fix that, for instance through curl.

**Comment by irishburlybear** [comment]
Where do I access curl from? That requires a prompt.. I'm on the solid
community server...

**Comment by Mitzi-Laszlo** [comment]
You can find more conversation on this subject here https://forum.solidproject.org/t/blocked-editing-to-public-folder/393
